dev-python/internetarchive/internetarchive-5.3.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="Lib and CLI for archive.org - for search, uploading, downloading, rename..."
```

Comment:
No ellipses or full stops at the end of DESCRIPTION. The upstream description reads:
> A Python and Command-Line Interface to Archive.org